app/controllers/tags_controller.rb

```diff
@@ -34,6 +34,11 @@ def destroy
     redirect_to tags_path, notice: t(".deleted")
   end
 
+  def destroy_all
+    Current.family.tags.destroy_all
+    redirect_back_or_to tags_path, notice: "All tags deleted"
+  end
```

Comment:
I'm still new to rails, but some of the functionality you can build out in a handful of lines of code is kinda insane

Reply:
The appeal of Rails for sure!